.github/workflows/main.yml

```diff
@@ -50,10 +50,58 @@ jobs:
           name: AppImage
           path: linuxdeploy*.AppImage*
 
+  container_build:
+    # The host should always be linux
+    runs-on: ubuntu-latest
+    name: ${{ matrix.BUILD_TYPE }} ${{ matrix.arch }}
+    strategy:
+      fail-fast: false
+      matrix:
+        BUILD_TYPE: [appimage, coverage]
+        distro:
+         - bullseye
```

Comment:
I think it would be better to do the build on Ubuntu bionic. It would run on more distros that way, if I'm not mistaken.